dependencies.props

```diff
@@ -77,6 +77,7 @@
       <BuildInfoPath>$(BaseDotNetBuildInfo)core-setup/$(DependencyBranch)</BuildInfoPath>
       <CurrentRef>$(CoreSetupCurrentRef)</CurrentRef>
     </RemoteDependencyBuildInfo>
+    <!--
     <RemoteDependencyBuildInfo Include="External">
```

Comment:
I suspect the better thing to do is to update the branch, similar to what we have done with standard.

Reply:
I think you're right. It's kind of weird though that these would have to point to the `master` channel for that to work currently, though. Do you think maybe we should fork the build-info txt files for each of these repos for 2.1 to make sure they'd be intentionally upgraded, never from a `master` build?

Reply:
To clarify, this is what I mean by forking: https://github.com/dagood/versions/commit/efa0e08fca0738189b0558aba29654412df97463

Then everything here just uses `$(DependencyBranch)`.

On the other hand, maybe `master` is misleading for these and it's ok to update from their `master` build-info channels?

Reply:
I don't think it is worth forking the versions repo artificially. 

I'm on the fence about just using master or disabling them like you have. On the one hand I don't think we will likely update these but on the other hand it might be interesting to see what is available to update. Given this is a release branch though I think we should probably error on the side of not updating, so I'm OK keeping these commented out like you have them.